packages/vite/src/node/__tests__/plugins/css.spec.ts

```diff
@@ -258,3 +261,142 @@ describe('convertTargets', () => {
     })
   })
 })
+
+describe('removePureCssChunks', () => {
```

Comment:
In the other comment where I suggested moving the `removePureCssChunks` code back into the Vite plugin, we can also remove this test for `removePureCssChunks` for now.